iree/compiler/Codegen/Common/TileAndDistributeToWorkgroupsPass.cpp

```diff
@@ -197,6 +197,8 @@ void TileAndDistributeToWorkgroupsPass::runOnOperation() {
   patterns.insert<TileAndDistributeLinalgOpsPattern,
                   IREE::LinalgExt::TiledOpInterfaceTilingPattern>(
       context, linalgTilingOptions, linalg::LinalgTransformationFilter(marker));
+  patterns.insert<linalg::ExtractSliceOfPadTensorSwapPattern>(
```

Comment:
I am not sure we want to use this pattern now. There is no `extract_slice` to do the swap with. Is this rebased on ToT?

Reply:
This is still needed. After we tiling the convolution/matmul op, there will be `tensor.extract_slice` ops generated inside the loop nest and  using `tensor.pad` op outside of the loop nest. This pattern then pulls in the `tensor.pad` op inside the loop nest. So effectively it's the "fusion" step for tile and fuse, but for `tensor.pad` op specifically. Without it, the `tensor.pad` op will be left outside of the loop nest.

Reply:
Can we move this out of this pass. This would affect all backends.... You can run this pattern in a separate pass on the SPIR-V side for now....